src/components/pip/pip.scss

```diff
@@ -47,9 +49,17 @@ $hideContainerGap: 5px;
     }
   }
 
+  .videoContainer {
```

Comment:
Please rename video to player as we will have also slides in the very near future

Reply:
done